.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Declare files that will always have LF line endings on checkout.

```

Comment:
Maybe leave a comment going more into detail why this is necessary, for when someone sees this several years from now and wonders why we have a random 2-line file.

Reply:
Fixed in a57c480866f30b32cb3248a25bec775e9d33b02f